src/js/edu-benefits/5490/config/form.js

```diff
@@ -406,8 +405,9 @@ const formConfig = {
                 'ui:options': {
                   hideIf: form => {
                     const status = _.get('highSchool.status', form);
-                    return status !== 'graduationExpected';
-                  }
+                    return status !== 'discontinued';
```

Comment:
Got it, I didn't realize those fields need to show for `graduated` still. This explanation helps a lot, thanks! I'll make those changes.